src/mbgl/style/update_parameters.hpp

```diff
@@ -42,6 +46,9 @@ class UpdateParameters {
 
     // TODO: remove
     Style& style;
+
+    const uint32_t fixedPrefetchZoom;
+    const uint32_t dynamicPrefetchZoom;
```

Comment:
What is the use case for having both fixed and dynamic prefetch? Which one takes precedence?